python/nano/test/neural-compressor/pytorch/test_trainer_quantize.py

```diff
@@ -72,7 +73,7 @@ def test_trainer_quantize_inc_ptq_compiled(self):
         x = next(train_loader_iter)[0]
 
         # Case 1: Default
-        qmodel = trainer.quantize(pl_model, self.train_loader)
+        qmodel = trainer.quantize(pl_model, self.train_loader, return_pl=False)
```

Comment:
The `Trainer.quantize` API is not updated according to the new design in https://github.com/intel-analytics/BigDL/issues/4360?

Reply:
Not yet. After this PR, we will have a separated PR to resolve it. All the usage in Chronos will be changed.